snuba/admin/views.py

```diff
@@ -1277,3 +1277,23 @@ def execute_job(job_id: str) -> Response:
 @check_tool_perms(tools=[AdminTools.DATABASE_CLUSTERS])
 def clickhouse_node_info() -> Response:
     return make_response(jsonify(get_node_info()), 200)
+
+
+@application.route("/clickhouse_system_settings")
+@check_tool_perms(tools=[AdminTools.DATABASE_CLUSTERS])
+def clickhouse_system_settings() -> Response:
+    try:
+        host = request.args.get("host")
+        port = request.args.get("port")
+        storage = request.args.get("storage")
+        if not all([host, port, storage]):
+            return make_response(
+                jsonify({"error": "Host, port, and storage are required"}), 400
+            )
+
```

Comment:
nit: I wonder if these lines need to be under the `try` block.

Reply:
I don't think they need to be, I'll pull them out